Documentation/Books/Manual/Programs/Arangodump/Examples.md

```diff
@@ -15,16 +15,22 @@ directory, you can use the parameter *--overwrite true* to confirm this:
     arangodump --output-directory "dump" --overwrite true
 
 _arangodump_ will by default connect to the *_system* database using the default
-endpoint. If you want to connect to a different database or a different endpoint,
-or use authentication, you can use the following command-line options:
+endpoint. To override the endpoint, or specify a different user, use one of the
+following commandline parameters:
```

Comment:
It's actually not just command line parameters, they could also be set via a config file. A neutral word would be _startup options_, see https://docs.arangodb.com/3.4/Manual/Administration/Configuration/

```suggestion
following startup options:
```